src/styles/footer.m.css

```diff
@@ -0,0 +1,8 @@
+@import './shared/variables.css';
+
+.footer {
+	background-color: var(--header-background-color);
+	bottom: 0;
+	position: absolute;
```

Comment:
Grid styling won't require things to be absolutely positioned, etc. the way they were in dgrid1. We should be able to remove bottom/position rules (see https://github.com/dojo/dgrid/issues/13)